### PR TITLE
Getter/Setter functions.

### PR DESCRIPTION
**Pull request by bhaskarmelkani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ snippet examples for your day to day workflow. Contributions are welcome!
 - [Maps](#maps)
 - [WeakMaps](#weakmaps)
 - [Promises](#promises)
+- [Getter/Setter functions](#getter-and-setter-functions)
 
 ## var versus let / const
```
Comment (Bereiche54) [inline README.md]: Please give me a merger

**Comment by DrkSephy** [comment]
I'm still thinking about adding this in or not, as it doesn't seem to be a high priority feature. Moreover, the code examples provided will need to be formatted accordingly to the other examples in the readme (such as leading space before an opening `{`). Does anyone else have some feedback on whether or not these features should be included?

File: README.md (hunk 1 above)
Comment (bhaskarmelkani) [inline README.md:25]: Did not get it ?

**Comment by bhaskarmelkani** [comment]
Fixed formatting.

**Comment by DrkSephy** [comment]
Thank you for updating your PR, could you please rebase and resolve any merge conflicts on your end so that I can merge this in? Thanks!

**Comment by bhaskarmelkani** [comment]
Done. Thanks

**Comment by DrkSephy** [comment]
Thanks! :+1:

**Pull request merged**